skimage/graph/__init__.py

```diff
@@ -1,3 +1,7 @@
+"""
+Utilities for Graph-based Image Segmentation, Object Detection and Network Analysis.
+This module provides functions for graph-based analysis of images and segmentations, e.g. path finding and cost evaluation, merging and cutting of graphs, or finding the closest centrality.
```

Comment:
Thanks for continuing on this.

We are using the [NumPyDoc standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) to format our docstrings. Could you
- Add a newline after the short summary
- Wrap each line to no more than 88 chars?

Also could you remove the capitalization of "Graph", "Image", and so forth in the first sentence except for the first letter of course?

Reply:
Thank you @lagru for guiding. I will make a new commit following your advice. I am grateful for your consideration.